tests/e2e_tests.py

```diff
@@ -39,20 +46,28 @@ def test_explicit_paths(link_localtime_mock, get_timezone_for_ip_mock):
             FAKE_TIMEZONE, zoneinfo_path, localtime_path,
         ),
     )
+    assert_true(
+        export_etc_timezone_mock.called_once_with(
+            FAKE_TIMEZONE, etc_timezone_path,
```

Comment:
[query] Are you sure this works? It looks like `etc_timezone_path` was not added to the arg list.